### ISSUE TITLE
Unicode characters not handled in recommendation response

### ISSUE THREAD

**Issue by aeyates** [DBCG/cqf-ruler#583]
Some of our CQL recommendations contain Unicode characters such as ≥ (https://www.compart.com/en/unicode/U+2265)

In CQF Ruler 0.4.1, this worked fine and we got the expected recommendations when executing the plan definition. However, in 0.5.0-SNAPSHOT these characters are converted to a question mark (e.g. "You recently had high blood pressure readings (? 130 mmHg systolic and/or ? 80 mmHg diastolic)")

I've attached a simple cql file and plan definition as a test. When I execute the plan definition in 0.4.1, this is the response:

```
{
    "cards": [
        {
            "summary": "This character is not handled: ≥",
            "indicator": "info",
            "detail": "None",
            "source": {}
        }
    ]
}
```

And when I execute against current master:

```
{
    "cards": [
        {
            "summary": "This character is not handled: ?",
            "indicator": "info",
            "detail": "None",
            "source": {}
        }
    ]

}
```

[simple.zip](https://github.com/DBCG/cqf-ruler/files/9246780/simple.zip)

**Comment by brynrhodes** [comment]
I can't be certain, but this may actually be addressed by #597

**Comment by c-schuler** [comment]
@aeyates The "applyCql" element was used for testing purposes and is not compliant with the CDS Hooks standard. The element should be removed from CDS Hooks requests and will result in an error in v0.5.1+

**Comment by c-schuler** [comment]
Addressed in #615
